packages/react-scripts/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## UNRELEASED
 
-_Nothing yet..._
+### Fixed
```

Comment:
Note: this will technically be a breaking change - no need to update it in the PR tho - you can just YOLO it after the release